types/hash-chain/index.d.ts

```diff
@@ -0,0 +1,94 @@
+// Type definitions for hash-chain 2.0
+// Project: https://github.com/emilbayes/hash-chain#readme
+// Definitions by: BendingBender <https://github.com/BendingBender>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+export = HashChain;
+
+/**
+ * Hash Chain implementation using Blake2b.
+ *
+ * @example
+ * import HashChain = require('hash-chain')
+ * import * as fs from 'fs'
+ *
+ * // Persistence
+ * var chain = // ...
+ * const fd = fs.open('filename')
+ * fs.write(fd, chain.offset, 0, 4)
+ * fs.write(fd, chain.chain)
+ * fs.close(fd)
+ *
+ * const data = fs.readFile('filename')
+ * const chain = new HashChain(data.subarray(4), data.readUInt32LE(0))
+ */
+declare class HashChain implements Iterable<Buffer> {
+    static readonly SEEDBYTES: number;
+    static readonly BYTES: number;
```

Comment:
```suggestion
    static readonly SEEDBYTES: 32;
    static readonly BYTES: 32;
```
we can be explicit (optionally)

Reply:
done